app/basicComponents/SideMenu.js

```diff
@@ -147,13 +157,14 @@ class SideMenu extends React.Component<Props, State> {
   }
 
   renderMenuElement = (item: SideMenuItem, index: number) => {
-    const { selectedItemIndex, loadingItemIndex } = this.props;
+    const { selectedItemIndex, loadingItemIndex, networkDisconnectedIndex } = this.props;
     const isDisabled = !item.path;
     return [
       item.hasSeparator && <Separator key="separator" />,
       <MenuItemWrapper key={item.text} onClick={isDisabled ? null : () => this.handleSelectEntry({ index })} isDisabled={isDisabled}>
         {index === selectedItemIndex && <SelectedMenuItemBorder />}
         <MenuItemIconWrapper>
+          {networkDisconnectedIndex === index && <NetworkDisconnected />}
```

Comment:
check param isConnected and then make decision based on network tab index, also add small comment as explanation for this

Reply:
ok